taggit/__init__.py

```diff
@@ -1,2 +1,2 @@
-VERSION = (6, 1, 0)
```

Comment:
Let's avoid bumping this up for now, we'll bump it up once we make a release

Reply:
Sounds good 🙌🏽